### PR TITLE
Syndicate Synthesis Specialist Midround Antag

### PR DESCRIPTION
**Pull request by Lyndomen**
## About the PR
Adds the Synthesis Specialist, along with their special shuttle to the event pool. The Synthesis Specialist operates on behalf of Interdyne, and seeks to peddle their chemistry knowledge and goods to the station- be it through poison, medicine, or anything in between. This role is intended to operate in a middle ground between chemistry, piracy, and negotiation. There are two per shuttle, and the event may only trigger once per shift. 

## Why / Balance
The Synthesis Specialist is balanced around needing to interact with the station to stay operable. They have no infinite power source, and must rely on plasma from trades to keep their relatively expensive ship powered. Additionally, they have a limited chemical supply further adding to their situation. They are ruled to stay attached to their ship, and should encourage a bit more creative antagonism.

## Technical details
YAML warrior 

## Media
image
image
image
image
image
image
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
Nothing seems broke. 

**Changelog**
:cl: Lyndomen & Fox
- add: Interdyne has begun sending Synthesis Specialists to bargain with the station!

### CODE REVIEW

File: Resources/Locale/en-US/deltav/synthesis/recruiter.ftl
Comment (DangerRevolution) [inline Resources/Locale/en-US/deltav/synthesis/recruiter.ftl:1]: "recruiter.ftl"

File: Resources/Prototypes/DeltaV/GameRules/events.yml [hunk 1]
```diff
@@ -207,3 +207,45 @@
...
+
+- type: entity
+  parent: BaseGameRule
+  id: SynthesisSpecialist
+  components:
+  - type: StationEvent
+    weight: 4
+    minimumPlayers: 20
+    maxOccurrences: 1
+    duration: null
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/GameRules/events.yml]:
```suggestion
    duration: 1 
```
Same effect, but no reason to make it null

File: Resources/Locale/en-US/deltav/synthesis/recruiter.ftl
Comment (Lyndomen) [reply Resources/Locale/en-US/deltav/synthesis/recruiter.ftl:1]: real ctrl v

**Comment by Avalon-Proto** [comment]
Should give them their own hypo-pens if they don't have them, so they can more easily deal with potential boarders

**Review by Axionyxx — APPROVED** [review]
Content 🤷

**Comment by Axionyxx** [comment]
For whatever reason going back to #general in the Discord opened right on lyndo saying "all my PRs are in review hell" with a link to this pr

**Comment by Lyndomen** [comment]
youre a real one

**Pull request merged**